components/scream/src/physics/shoc/shoc_error_function_impl.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef SHOC_ERROR_FUNCTION_IMPL_HPP
+#define SHOC_ERROR_FUNCTION_IMPL_HPP
+
+#include "shoc_functions.hpp" // for ETI only but harmless for GPU
+
+namespace scream {
+namespace shoc {
+
+/*
+ * Implementation of shoc error_function. Clients should NOT
+ * #include this file, but include shoc_functions.hpp instead.
+ */
+
+template<typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>::error_function(const Scalar &input,
+                                    Scalar       &output)
+{
+  output = std::erf(input);
```

Comment:
I think this might not compile/work on gpu. Or, at the very least, not give bfb results with the host impl. I think you might need something similar to `cxx_sqrt` for erf.

Reply:
I think you are right. It compiles on GPU, but I believe my test doesn't test bfb